### PR TITLE
Update Accumulo Tour Source Code

### PR DESCRIPTION
**Pull request by slackwinner**
Minor coding, grammar, and comment improvements have been made
within the "tour" folder to enhance the Accumulo Tour experience for 
new users.

### CODE REVIEW

**Comment by slackwinner** [comment]
One thing I would note is that during the tour I encountered this warning message: 

WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.hadoop.security.authentication.util.KerberosUtil (file: ...)
WARNING: Please consider reporting this to the maintainers of org.apache.hadoop.security.authentication.util.KerberosUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

This doesn't break the tour but the warning message appears if users are executing the code with Java versions 9 and above.

File: tour/authorizations-code.md
Comment (ctubbsii) [inline tour/authorizations-code.md:9]: :+1: These make things a little more consistent. Normally, this wouldn't matter, but since this is intended to be example code that users following along the tour will use, it can help make the tour more presentable.

File: tour/authorizations-code.md
Comment (ctubbsii) [inline tour/authorizations-code.md:44]:
```suggestion
    // Create a second client for the commissioner user and output all the rows visible to them.  
```

File: tour/authorizations-code.md
Comment (ctubbsii) [inline tour/authorizations-code.md:50]: These are good changes, but it would be nice if they matched our coding style, including using two spaces instead of tab characters. If you are using Eclipse, you can import [our code style](https://github.com/apache/accumulo/blob/main/contrib/Eclipse-Accumulo-Codestyle.xml) to format the code.

File: tour/authorizations.md
Comment (ctubbsii) [inline tour/authorizations.md:5]: It's probably fine to preserve the uppercase for these, since we literally mean `String` in Java. It might help to use syntax highlighting in Markdown whenever we reference a specific class (singular or plural), such as:

```suggestion
[Authorizations] are a set of `String`s that enable a user to read protected data. Users are granted authorizations
```

File: tour/basic-read-write.md
Comment (ctubbsii) [inline tour/basic-read-write.md:30]:
```suggestion
        // Note: A Scanner is an extension of java.lang.Iterable so it will traverse through the scanner's range.
        // In this case, since no range was set on the scanner, it will traverse the entire table.
```

File: tour/batch-scanner.md
Comment (ctubbsii) [inline tour/batch-scanner.md:4]: This is another one where we literally mean objects of type `Key`, so uppercase is okay. I could go either way on these things, like `String` and `Key`, since their functional name and their proper name are the same and interchangeable. It might be nice if we were a bit more consistent on case, and syntax highlighting (see my other comment regarding `String` for the suggestion to use syntax highlighting).

**Review by ctubbsii — CHANGES_REQUESTED** [review]
Thanks for the contribution, @slackwinner ! These are nice tweaks to the tour that will make it more readable. I made a few suggestions, however, that you may wish to consider.

The only change I will insist on is the removal of tab characters in favor of spaces.

If you aren't already aware, any additional changes you make to your branch will automatically show up in this PR. You can just add new commits, and we'll squash them all together into one larger commit once it's ready for merging.

File: tour/authorizations-code.md
Comment (slackwinner) [reply tour/authorizations-code.md:50]: Thanks for pointing that out. Didn't know we had a specific coding style for this repo until now. I'll change the tab characters to two spaces so the style stays consistent going forward.

File: tour/batch-scanner.md
Comment (slackwinner) [reply tour/batch-scanner.md:4]: I see your point. At the time of committing I was focused on grammar but didn't take into consideration that we were referring to objects. I'll apply the fixes first thing tomorrow morning.

File: tour/authorizations-code.md
Comment (ctubbsii) [reply tour/authorizations-code.md:50]: Yeah, our code is typically formatted to conform to our style by the formatter-maven-plugin when we build/compile, but code on the website isn't built, since it's more documentation than code. So, it doesn't get automatically formatted.

We're not strict about the style in areas where it's not automatically enforced, but it is nice to be somewhat consistent, if we can. Also, I think it's generally understood that we might need to break our style conventions to improve readability (especially in documentation). However, spaces instead of tabs is the one area where we're pretty consistent across all our repos/projects, using them only when necessary (such as in a Makefile).

**Comment by slackwinner** [comment]
@ctubbsii Based on our conversation yesterday, I've applied additional changes to the tour files. The main changes include swapping the tab characters with two spaces, adding syntax highlighting, and fixing the space formatting to reflect our coding style. Please let me know if more changes are needed.

File: tour/ranges-splits.md
Comment (ctubbsii) [inline tour/ranges-splits.md:35]: "Tablet Servers" isn't a class name. If your intent is to highlight only class names, then we could highlight it as `TabletServer`s, but in this case, I don't think it should be highlighted at all, because it makes sense to use the semi-proper name "Tablet Server" rather than the class name, "TabletServer".

```suggestion
When working with large amounts of data across many Tablet Servers, a simple `Scanner` might not do the trick. Next lesson we learn about the power of 
```

File: tour/ranges-splits.md
Comment (ctubbsii) [inline tour/ranges-splits.md:20]: The idea I had for highlighting was to avoid including the pluralizing `s` inside the highlighted portion, as in:

```suggestion
By default, a table will get split into `Tablet`s on row boundaries, guaranteeing an entire row to be on one `Tablet Server`.  We have the ability to 
```

That way, it's clear that the `s` isn't part of the class name and exists only to pluralize.

File: tour/batch-scanner-code.md
Comment (ctubbsii) [inline tour/batch-scanner-code.md:27]: This code block could be formatted, especially to exclude tabs in favor of space characters.

File: tour/basic-read-write.md
Comment (ctubbsii) [inline tour/basic-read-write.md:37]: I think you may have accidentally copied and pasted this twice.

File: tour/authorizations.md
Comment (ctubbsii) [inline tour/authorizations.md:43]:
```suggestion
    .as("commissioner", "gordonrocks").build()) {
  // your code here
}
```

File: tour/authorizations-code.md
Comment (ctubbsii) [inline tour/authorizations-code.md:62]: This code block could be formatted, and should not include tab characters.

**Review by ctubbsii — CHANGES_REQUESTED** [review]
Hi @slackwinner . I've taken another pass. Thanks for working to improve the tour. I have a few suggestions. Some of these suggestions may apply to other parts of your pull request. So, if you update to incorporate a suggestion in one place, please check to see if it might apply elsewhere in your changes. Thanks!

File: tour/ranges-splits.md
Comment (slackwinner) [reply tour/ranges-splits.md:35]: Yeah I originally thought you wanted to highlight all the class names. Looking back now, I agree that we should get rid of the redundant highlighted words as. I will go through the tour code and get rid of some of the highlighted words.

File: tour/authorizations-code.md
Comment (slackwinner) [reply tour/authorizations-code.md:62]: I believe what happened here is that I accidentally left out the this file when I was committing the second time. The version I have right now gets rid of the tab characters. Will send up a third commit soon.

**Comment by ctubbsii** [comment]
Thanks for the PR @slackwinner . If you wish to be added as a contributor to https://accumulo.apache.org/people/ , please open a new pull request to add yourself at https://github.com/apache/accumulo-website/edit/main/pages/people.md and leave a reference to `apache/accumulo-website#255` in your commit log.

If you intend to be a regular contributor to Accumulo projects, please consider subscribing to our developer mailing list (https://accumulo.apache.org/contact-us/) and introducing yourself. :smiley_cat:

**Pull request merged**
